### PR TITLE
[workloads] Add net current to current names to make them match in net10

### PR DESCRIPTION
**Pull request by lewing**

### CODE REVIEW

File: src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest/WorkloadManifest.json.in [hunk 1]
```diff
@@ -254,119 +254,119 @@
         "osx-arm64": "Microsoft.NETCore.App.Runtime.AOT.osx-arm64.Cross.android-arm64"
```
Comment (joeloff) [inline src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest/WorkloadManifest.json.in:254]: Does the alias need the `{NetVersion}` too?

Comment (lewing) [reply src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest/WorkloadManifest.json.in:254]: probably need to make sure we add aliases where there weren't before and I need to make sure I understand what is going on with the win runtime packs

**Review by steveisok — APPROVED** [review]
This looks good, but we should run an official build as well to really check.

File: src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest/WorkloadManifest.json.in (hunk 1 above)
Comment (marcpopMSFT) [reply src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest/WorkloadManifest.json.in:287]: I don't think we had the netversion in any aliases before, it was just the pack names themselves (as this was for VS authoring)

**Comment by marcpopMSFT** [comment]
@lewing This only adds the net9 labels. Will there be a process to port the PR you have for net8 to add those same labels into main (as that PR won't port without fixing conflicts)?

**Comment by lewing** [comment]
> @lewing This only adds the net9 labels. Will there be a process to port the PR you have for net8 to add those same labels into main (as that PR won't port without fixing conflicts)?

We discussed this offline but I'll follow up here too incase others have the same question. The net8 labels are already changed in main, the issue is that before this pr the names for the release branches wouldn't have the embedded change

**Comment by MichalStrehovsky** [comment]
@lewing I don't think I've seen a successful official build since this merged. First there were all sorts of things wrong with it, but these days it stabilized to:

```
D:\a\_work\1\s\src\workloads\workloads.csproj(147,5): error : System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.NET.Runtime.LibraryBuilder.Sdk' was not present in the dictionary.
D:\a\_work\1\s\src\workloads\workloads.csproj(147,5): error :    at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
D:\a\_work\1\s\src\workloads\workloads.csproj(147,5): error :    at Microsoft.DotNet.Build.Tasks.Workloads.CreateVisualStudioWorkload.ExecuteCore() in /_/src/Microsoft.DotNet.Build.Tasks.Workloads/src/CreateVisualStudioWorkload.wix.cs:line 221
D:\a\_work\1\s\src\workloads\workloads.csproj(147,5): error :    at Microsoft.DotNet.Build.Tasks.Workloads.VisualStudioWorkloadTaskBase.Execute() in /_/src/Microsoft.DotNet.Build.Tasks.Workloads/src/VisualStudioWorkloadTaskBase.wix.cs:line 105
##[error]src\workloads\workloads.csproj(147,5): error : (NETCORE_ENGINEERING_TELEMETRY=Build) System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.NET.Runtime.LibraryBuilder.Sdk' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
```

I'm not sure if this PR is to blame because there were so many other issues around the time it merged, but it looks likely.

**Comment by lewing** [comment]
> @lewing I don't think I've seen a successful official build since this merged. First there were all sorts of things wrong with it, but these days it stabilized to:
> 
> ```
> D:\a\_work\1\s\src\workloads\workloads.csproj(147,5): error : System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.NET.Runtime.LibraryBuilder.Sdk' was not present in the dictionary.
> D:\a\_work\1\s\src\workloads\workloads.csproj(147,5): error :    at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
> D:\a\_work\1\s\src\workloads\workloads.csproj(147,5): error :    at Microsoft.DotNet.Build.Tasks.Workloads.CreateVisualStudioWorkload.ExecuteCore() in /_/src/Microsoft.DotNet.Build.Tasks.Workloads/src/CreateVisualStudioWorkload.wix.cs:line 221
> D:\a\_work\1\s\src\workloads\workloads.csproj(147,5): error :    at Microsoft.DotNet.Build.Tasks.Workloads.VisualStudioWorkloadTaskBase.Execute() in /_/src/Microsoft.DotNet.Build.Tasks.Workloads/src/VisualStudioWorkloadTaskBase.wix.cs:line 105
> ##[error]src\workloads\workloads.csproj(147,5): error : (NETCORE_ENGINEERING_TELEMETRY=Build) System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.NET.Runtime.LibraryBuilder.Sdk' was not present in the dictionary.
>    at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
> ```
> 
> I'm not sure if this PR is to blame because there were so many other issues around the time it merged, but it looks likely.

That error was fixed a while back, what build are you looking at?

**Comment by MichalStrehovsky** [comment]
> > @lewing I don't think I've seen a successful official build since this merged. First there were all sorts of things wrong with it, but these days it stabilized to:
> > ```
> > D:\a\_work\1\s\src\workloads\workloads.csproj(147,5): error : System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.NET.Runtime.LibraryBuilder.Sdk' was not present in the dictionary.
> > D:\a\_work\1\s\src\workloads\workloads.csproj(147,5): error :    at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
> > D:\a\_work\1\s\src\workloads\workloads.csproj(147,5): error :    at Microsoft.DotNet.Build.Tasks.Workloads.CreateVisualStudioWorkload.ExecuteCore() in /_/src/Microsoft.DotNet.Build.Tasks.Workloads/src/CreateVisualStudioWorkload.wix.cs:line 221
> > D:\a\_work\1\s\src\workloads\workloads.csproj(147,5): error :    at Microsoft.DotNet.Build.Tasks.Workloads.VisualStudioWorkloadTaskBase.Execute() in /_/src/Microsoft.DotNet.Build.Tasks.Workloads/src/VisualStudioWorkloadTaskBase.wix.cs:line 105
> > ##[error]src\workloads\workloads.csproj(147,5): error : (NETCORE_ENGINEERING_TELEMETRY=Build) System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.NET.Runtime.LibraryBuilder.Sdk' was not present in the dictionary.
> >    at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > I'm not sure if this PR is to blame because there were so many other issues around the time it merged, but it looks likely.
> 
> That error was fixed a while back, what build are you looking at?

Sorry, I've lost it. We have brand new official build issues now but official build hasn't really worked in a week so I'm just trying to figure out what are all the things that need to be fixed in parallel. Looks like this one shouldn't be an issue anymore then.

**Comment by lewing** [comment]
> > > @lewing I don't think I've seen a successful official build since this merged. First there were all sorts of things wrong with it, but these days it stabilized to:
> > > ```
> > > D:\a\_work\1\s\src\workloads\workloads.csproj(147,5): error : System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.NET.Runtime.LibraryBuilder.Sdk' was not present in the dictionary.
> > > D:\a\_work\1\s\src\workloads\workloads.csproj(147,5): error :    at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
> > > D:\a\_work\1\s\src\workloads\workloads.csproj(147,5): error :    at Microsoft.DotNet.Build.Tasks.Workloads.CreateVisualStudioWorkload.ExecuteCore() in /_/src/Microsoft.DotNet.Build.Tasks.Workloads/src/CreateVisualStudioWorkload.wix.cs:line 221
> > > D:\a\_work\1\s\src\workloads\workloads.csproj(147,5): error :    at Microsoft.DotNet.Build.Tasks.Workloads.VisualStudioWorkloadTaskBase.Execute() in /_/src/Microsoft.DotNet.Build.Tasks.Workloads/src/VisualStudioWorkloadTaskBase.wix.cs:line 105
> > > ##[error]src\workloads\workloads.csproj(147,5): error : (NETCORE_ENGINEERING_TELEMETRY=Build) System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.NET.Runtime.LibraryBuilder.Sdk' was not present in the dictionary.
> > >    at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > I'm not sure if this PR is to blame because there were so many other issues around the time it merged, but it looks likely.
> > 
> > 
> > That error was fixed a while back, what build are you looking at?
> 
> Sorry, I've lost it. We have brand new official build issues now but official build hasn't really worked in a week so I'm just trying to figure out what are all the things that need to be fixed in parallel. Looks like this one shouldn't be an issue anymore then.

There are a lot of people trying to get the official build passing again, hitting a lot of infrastructure problems at the moment.

**Pull request merged**